code/wdb/wdbstandard.q

```diff
@@ -0,0 +1,22 @@
+// Get the relevant WDB attributes
+.proc.getattributes:{default:`date`tables`procname!(.wdb.currentpartition;tables[];.proc.procname);
+    / get all cols that contains date (of type "pdz")
+    timecolumns:1!{tcols:exec c from meta value x where t in"pdz";
+        (enlist[`tablename]!enlist x),
+            /functional select to get the min value (defaults to `timestamp$.z.d for starttimestamp)
+            enlist[`timecolumns]!enlist?[x;();();
+                tcols!(enlist,/:enlist each($;enlist`timestamp),/:enlist each((?),/:enlist each(=;0W),/:mtcols),'`.z.d,'mtcols:enlist each min,/:tcols),\:0Wp]}each tables[`.];
+    / update date attribute for .gw.partdict and .gw.attributesrouting
+    default[`date]:asc default[`date]union first[d]+til 1+last deltas d:exec(min;max)@\:distinct`date$raze[value each timecolumns][;0]from timecolumns;
+    if[.ds.datastripe;
```

Comment:
I thought wdb wouldn't be running if data striping was turned on? Has that changed?

Reply:
I think this is redundant code from before when the tailer and wdb were one in the same so it can be safely removed.